tools/unsupervised_dataset/Crawler/README.md

```diff
@@ -0,0 +1,11 @@
+# SQL Crawler
+
+This directory contains the code to run a universal, unsupervised SQL web crawler. The user provides a starting target URL from which to begin crawling, and has the option to set the maximum depth or size of the crawler.
+
+## Usage
+To run the crawler, run the following command:
+
+```
+python3 run_crawler.py <starting URLs>
```

Comment:
nit: add a comment on "URL should be  separated by comma or space/something else" ?